### ISSUE TITLE
AnimationTreePlayer blending normal tracks

### ISSUE THREAD

**Issue by FEDE0D** [godotengine/godot#2299]
AnimationTreePlayer works fine when blending the animations that uses transform tracks. Unfortunately these tracks are not easy to modify (there is no auto insert track or insert key as in normal tracks) and there's no way to use this tracks in 2d (or at least I think so).

I've made a project testing the blending of two animations (horizontal & vertical) using normal and transform tracks in 2d and 3d.

I don't know if this conversion from transform to normal tracks blending is gonna take a while (I think the transform tracks are older than the normal tracks and the Tween utility  ?), but this could be a nice thing to have in the animation modules. 

Basically this:
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/BYZuVQlFpko/0.jpg)](http://www.youtube.com/watch?v=BYZuVQlFpko)

Here's the project: https://dl.dropboxusercontent.com/u/274954519/animation-tree-tests.zip

**Comment by FEDE0D** [comment]
This is still not implemented/fixed on 2.0 alpha 20151030

**Comment by FEDE0D** [comment]
Can we change the label to bug or something more critical than feature request?
Its a must have, and it should be already working.

**Comment by JoshuaGrams** [comment]
And here's a Windows build if people want to hammer on it.

http://arestlessmind.org/2016/02/17/godot.windows.tools.exe
